numismatic/events.py

```diff
@@ -28,10 +30,24 @@ class Heartbeat(Event):
 
 
 @attr.s(slots=True)
-class Trade(Event):
+class PriceUpdate(Event):
     exchange = attr.ib(convert=str)
+    # FIXME: symbol should be split into asset and currency
```

Comment:
This seems like a very important FIXME.

Reply:
Indeed it is. Symbol name is confusing and changes to much from exchange to exchange. Best to keep them separate